Source/PINCacheConfigurationInternal.h

```diff
@@ -0,0 +1,17 @@
+//
+//  PINCacheConfigurationInternal.h
+//  PINCache
+//
+//  Created by Vladimir Solomenchuk on 3/19/20.
+//  Copyright © 2020 Pinterest. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "PINCacheConfiguration.h"
+
+extern BOOL PINCacheIsActiveExperimentalFeature(PINCacheExperimentalFeatures option);
+
+@interface PINCacheConfigurationManager : NSObject
+// only for tests
++ (void) test_resetWithConfiguration:(PINCacheExperimentalFeatures) features;
```

Comment:
Are the tests ever called?

Reply:
not yet. It will be used in unit tests, when you want to test experimental functionality.